azure-kusto-data/azure/kusto/data/_version.py

```diff
@@ -1,2 +1,2 @@
 """Version file."""
-VERSION = "0.0.43"
+VERSION = "0.0.44"
```

Comment:
this is only done when releasing a version.. no need

Reply:
👍